global_helpers/panther_iocs.py

```diff
@@ -163,6 +163,39 @@
     "shscrypto.net",
 }
 
+CRYPTO_MINING_PORTS = {
```

Comment:
Do you have a reference link you could add here?

Reply:
Addressed in https://github.com/panther-labs/panther-analysis/pull/972/commits/cada7ed5a06610058328e58012826d8eccf39c1a.